packages/server/tests/acceptance/erc20.spec.ts

```diff
@@ -301,7 +301,7 @@ describe('@erc20 Acceptance Tests', async function () {
                                             // eth_estimateGas gets called by ethers
                                             // so we need to catch the error and check that the reason is the expected one,
                                             // in addition to validating the CALL_EXCEPTION   
-                                            expect(extractRevertReason(e.error.reason)).to.be.equal('ERC20: transfer amount exceeds balance');                                            
+                                            expect(extractRevertReason(e.error.reason)).to.be.equal('ERC20: insufficient allowance');
```

Comment:
The mirror node fix for this was merged, and it looks as if it will be in 0.85 of the mirror node.  Once the mirror node fix is in testnet the revert reason message will need to be set back to 'ERC20: transfer amount exceeds balance'